components/automate-chef-io/content/docs/reports.md

```diff
@@ -74,8 +76,11 @@ Please note the only supported filters for deep filtering are:
 
 ### Waivers
 
-Waived status of nodes can be seen throughout the pages where status is listed. Users can hover over waived controls' status icon in the controls list to view more details about the waiver applied to the control.
-Use of waivers is configured using Chef InSpec: https://www.inspec.io/docs/reference/waivers/
+A node's waived status appears if applicable in displays where a node's status appears in Chef Automate. 
+The Compliance Reports overview displays the node count and history of waived nodes, and the count and history of waived controls.
+Hover over the control's Waived icon in the Controls list to view more details about the waiver applied to the control.
```

Comment:
should we say "controls list view's node status column" ?

Reply:
@mjingle ^^

Reply:
@vjeffrey The icon technically appears for each individual control in Controls, but thank you for making me realize this paragraph needed further expansion!